app/models/release.rb

```diff
@@ -0,0 +1,21 @@
+class Release < ApplicationRecord
+  TYPES = %w{
+    limited_scope
+  }.freeze
+
+  belongs_to :friend
+  belongs_to :user # Validation below that this user is a lawyer
+
+  mount_uploader :release_form, ReleaseFormUploader
+
+  validates :release_form, :friend, presence: true
+  validates :category, presence: true, inclusion: { in: TYPES }, uniqueness: { scope: [:friend_id, :user_id] }
+  validate :user_is_lawyer
+
+  private
+
+  def user_is_lawyer
+    return unless user
+    errors.add(:user, 'must be lawyer') unless user.volunteer_type == 'lawyer'
```

Comment:
This is one of the super, duper confusing things the current state of lawyers in the codebase.  The volunteer_type 'lawyer' doesn't have any bearing on being a 'remote_clinic_lawyer'.  We have a boolean 'remote_clinic_lawyer"....so if we check that is true instead, we'll be golden.